modules/core/http/AgsHttp.swift

```diff
@@ -0,0 +1,21 @@
+import AeroGearHttp
+import Foundation
+
+public protocol Request {
+}
+
+/**
+ * Wrapper class used for network requests
+ */
+public class AgsHttp {
```

Comment:
Consider use struct instead of class

Reply:
I'm going to extend that soon - it's just not clear how networking layer may work now since we do not have any service.